internal/ansible/ansible.go

```diff
@@ -358,6 +359,14 @@ func (r *Runner) Run() (*exec.Cmd, io.Reader, error) {
 	dc.Stdout = stdoutWriter
 	dc.Stderr = stderrWriter
 
+	// let the command shut down gracefully
+	dc.Cancel = func() error {
+		return dc.Process.Signal(os.Interrupt)
+	}
+	// if it doesn't respond to the SIGINT within 10s,
+	// it's going to be forcefully shut down with SIGKILL
+	dc.WaitDelay = 10 * time.Second
```

Comment:
Just curious if it makes sense to allow the value of WaitDelay configurable.

Reply:
I'd leave it non-configurable until someone comes up with a use-case or asks for it, but that said, can totally add the flag - you're the boss :)